ShowQuestion.go

```diff
@@ -0,0 +1,35 @@
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/manifoldco/promptui"
+)
+
+func ShowQuestion(message string) bool {
```

Comment:
I extended this with 
`func ShowQuestion(message string, defaultValueFlag bool) bool`
then use that flag to set `Default` to either `y` or `n` so I can have different question types.